src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
> It looks like now adding a jit specific config value requires adding at two places: src/inc/clrconfigvalues.h and src/jit/jitconfigvalues.h

The former is not necessary unless the value is also needed by the EE or the value is needed in JITs that are not RyuJIT. The EE's implementation JIT interface will translate the given value name into a `CLRConfig::ConfigX` key.

> Is there anything in the system to detect discrepancy between the type of value added in clrconfigvalues.h and jitconfigvalues.h?

No. The values that remain in clrconfigvalues.h are only there for compatibility with other JIT builds.